ArcGISRuntimeSDKQt_CppSamples/Layers/DisplayAnnotation/README.md

```diff
@@ -0,0 +1,37 @@
+# Display Annotation
+
+Display annotation from a feature service URL.
+![](screenshot.png)
+
+## Use case
+Annotation is useful for displaying text that you don't want to move or resize when the map is panned or zoomed (unlike labels which will move and resize). You can use annotation to place text at a fixed size, position, orientation, font, and so on. You may choose to do this for cartographic reasons or because the exact placement of the text is important.
+
+## How to use the sample
+Pan and zoom to see names of waters and burns in a small region of Scotland.
+
+## How it works
+1. Create a `Map` with a light gray canvas and a viewpoint near the data.
+2. Create an `AnnotationLayer` from a feature service URL.
+3. Add both layers to the operational layers of the map and add it to a `MapView`.
+
+## Relevant API
+ - AnnotationLayer
+ - FeatureLayer
+ - ServiceFeatureTable
+
+## About the data
+
+Data derived from [Ordnance Survey OpenRivers](https://www.ordnancesurvey.co.uk/business-government/products/open-map-rivers). Contains OS data &copy; Crown copyright and database right 2018.
```

Comment:
is the `&copy;` supposed to be here? I'm unsure what it means

Reply:
Copyright symbol!

Reply:
how come this renders properly in the QML readme but not the C++ readme?